http2_server.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+import socket
+import email.utils
+import mimetypes
+import os
+
+
+ROOT = os.path.join(os.getcwd(), 'webroot')
+
+
+def response_ok(resolved_uri):
+    """Returns response code with date and headers"""
+    response_code = "HTTP/1.1 200 OK"
+    headers = "Content-Type: {}".format(resolved_uri[0])
+    date = "Date: {}".format(email.utils.formatdate(usegmt=True))
+    response = "{}\r\n{}\r\n{}\r\n\r\n{}".format(
+        response_code, date, headers, resolved_uri[1])
+    return response
```

Comment:
This is correct, but i'd still *so* much rather see `"\r\n".join([list, of, response, lines])`